PR: https://github.com/odoo/odoo/pull/103682

From: 5f006544918216d1cf9504264b7c30e28176ab9d
From: maximilien(malb)
Date: 2022-10-21 12:06:16

Breaking data model changes score: 6.4, change matches:
-    match_partner = fields.Boolean(string='Partner should be set', tracking=True,
+    match_partner = fields.Boolean(string='Partner is Set', tracking=True,
-    match_partner_ids = fields.Many2many('res.partner', string='Only Those Partners',
+    match_partner_ids = fields.Many2many('res.partner', string='Matching partners',
-    match_partner_category_ids = fields.Many2many('res.partner.category', string='Only Those Partner Categories',
+    match_partner_category_ids = fields.Many2many('res.partner.category', string='Matching categories',

Total Changes: 6

[FIX] account: reconcile model display improvement

Most of the dropdown of the account reconcile model view were weirdly displayed, the case was too small for the text, by removing the class oe_inline most of the dropdown were fixed, and for the other just a slight modification on the width did the job.

Also on the left column the two first line were not aligned with the rest, same for the right column on the first line, now it's fixed but I had to put the invisible in the field

Other thing, the counterpart entries were too much on the left of the page due to a useless group, it's now removed

Also change some display name and added the translation

closes odoo/odoo#103682

Task-id: 3000402
X-original-commit: 487cc0959991d804a48f44b914540786a85f180f
Signed-off-by: Laurent Smet <las@odoo.com>
Signed-off-by: Maximilien La Barre (malb) <malb@odoo.com>

================================= pseudo patch: =================================

--- a/addons/account/models/account_reconcile_model.py
+++ b/addons/account/models/account_reconcile_model.py
@@ -338,11 +338,11 @@ class AccountReconcileModel(models.Model):
         tracking=True,
         help="The sum of total residual amount propositions and the statement line amount allowed gap type.",
     )
-    match_partner = fields.Boolean(string='Partner should be set', tracking=True,
+    match_partner = fields.Boolean(string='Partner is Set', tracking=True,
         help='The reconciliation model will only be applied when a customer/vendor is set.')
-    match_partner_ids = fields.Many2many('res.partner', string='Only Those Partners',
+    match_partner_ids = fields.Many2many('res.partner', string='Matching partners',
         help='The reconciliation model will only be applied to the selected customers/vendors.')
-    match_partner_category_ids = fields.Many2many('res.partner.category', string='Only Those Partner Categories',
+    match_partner_category_ids = fields.Many2many('res.partner.category', string='Matching categories',
         help='The reconciliation model will only be applied to the selected customer/vendor categories.')
 
     line_ids = fields.One2many('account.reconcile.model.line', 'model_id')
